### ISSUE TITLE
TYPO3 13LTS

### ISSUE THREAD

**Issue by creationell** [bithost-gmbh/pdfviewhelpers#249]
Hello, are you working on a release for the new TYPO3 13? Is there a time shedule?

**Comment by maechler** [comment]
Hi @creationell, we are not yet actively working on it, but we will definitely add support for TYPO3 13 LTS! I will probably tackle this towards the end of the year, however if you already feel like trying it out and reporting potential issues, you are very welcome to do so! I don't know yet if it even requires changes to the extension.

**Comment by creationell** [comment]
Thank you for your quick reply!

**Comment by maechler** [comment]
Unfortunately the tests are not yet running through and I won't have much time in the next month to look into the new setup needed. But whoever wants to try it already can use this branch: 
https://github.com/bithost-gmbh/pdfviewhelpers/tree/typo3-v13

It seems that most things should already work: https://github.com/bithost-gmbh/pdfviewhelpers/pull/250#issue-2605489188

**Comment by terminal8-nz** [comment]
Any news on this issue? Is there a time schedule for the release?

**Comment by maechler** [comment]
@terminal8-nz No, I'm afraid. I didn't have the time so far, I want to have a look again in the next 1-2 months. If you have time to check the failing tests or do some other testing, that would be appreciated.

https://github.com/bithost-gmbh/pdfviewhelpers/tree/typo3-v13

If you need it already now, you could try installing it from the branch using version `dev-typo3-v13` and report potential issues here.

https://packagist.org/packages/bithost-gmbh/pdfviewhelpers#dev-typo3-v13

**Comment by froemken** [comment]
> Unfortunately the tests are not yet running

and 

> If you have time to check the failing tests

For these cases I have just created a nearly +2000 rows patch. Tests are running again. Happy merging and...oh sorry...reviewing ;-)

Stefan

**Comment by maechler** [comment]
I just released a new version `v3.1.0` supporting TYPO3 v13, thank you all for your help!

**Comment by froemken** [comment]
@maechler Done. I'm a little bit frustrated, but it's OK. Target achieved...a new version for TYPO3 13 is on TER, so I'm happy again. Yeah. Thank you for finding time for that upgrade.
